agent/agent.go

```diff
@@ -1114,21 +1115,6 @@ func newConsulConfig(runtimeCfg *config.RuntimeConfig, logger hclog.Logger) (*co
 	if runtimeCfg.ACLMasterToken != "" {
 		cfg.ACLMasterToken = runtimeCfg.ACLMasterToken
 	}
-	if runtimeCfg.ACLTokenTTL != 0 {
-		cfg.ACLTokenTTL = runtimeCfg.ACLTokenTTL
-	}
-	if runtimeCfg.ACLPolicyTTL != 0 {
-		cfg.ACLPolicyTTL = runtimeCfg.ACLPolicyTTL
-	}
-	if runtimeCfg.ACLRoleTTL != 0 {
-		cfg.ACLRoleTTL = runtimeCfg.ACLRoleTTL
-	}
-	if runtimeCfg.ACLDefaultPolicy != "" {
-		cfg.ACLDefaultPolicy = runtimeCfg.ACLDefaultPolicy
-	}
-	if runtimeCfg.ACLDownPolicy != "" {
-		cfg.ACLDownPolicy = runtimeCfg.ACLDownPolicy
-	}
```

Comment:
I notice that `ACLDisabledTTL` is missing from this list. I believe that means that the `acl.disabled_ttl` that we set in `agent/config.NonUserSource` (`120s`) is completely ignored. The value that is actually used is the one from `consul.DefaultConfig`, which is set to `30s`. I guess that's probably not a huge problem, just a potential source of confusion.

I can look at cleaning that up in a follow up.

Reply:
PR'ed this cleanup in #10814